pages/08.scr/default.en.md

```diff
@@ -0,0 +1,42 @@
+---
+title: SysEleven Container Registry
+page-toc:
+    active: false
+body_classes: 'title-center title-h1h2'
+---
+
+[SysEleven Container Registry (SCR)](https://scr.syseleven.de) is the managed registry for containers and Helm charts from [SysEleven](https://www.syseleven.de) - a Berlin based managed hosting company.
```

Comment:
Harbor is a "(container) image registry" not a "container registry", it stores images not containers.
The reader is already on our documentation. We do not need to explain what SysEleven is.